### PR TITLE
trigger matching

### PR DESCRIPTION
**Pull request by jshlee**
added patTriggerObjectStandAlones for trigger matching
plus updates to h2mu analysis

### CODE REVIEW

File: CatAnalyzer/interface/AnalysisHelper.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#include "CATTools/DataFormats/interface/Particle.h"
+
+#include "TString.h"
+#include "TLorentzVector.h"
+
+class AnalysisHelper {
+
+ public:
+  static float deltaR(float e1,float p1,float e2,float p2);
+  static float deltaR(reco::LeafCandidate & l1, reco::LeafCandidate & l2){return deltaR(l1.eta(),l1.phi(),l2.eta(),l2.phi());}
```
Comment (jhgoh) [inline CatAnalyzer/interface/AnalysisHelper.h]: There are standard deltaR function in the CMSSW. why don't you use it?

Comment (jshlee) [inline CatAnalyzer/interface/AnalysisHelper.h]: yep ok

File: CatAnalyzer/plugins/h2muAnalyzer.cc [hunk 2]
```diff
@@ -348,51 +405,19 @@ int h2muAnalyzer::JetCat_GC(float mu1_eta, float mu2_eta)
...
-void 
-h2muAnalyzer::endLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&)
-{
-}
-
-// ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
-void
-h2muAnalyzer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+void h2muAnalyzer::beginJob(){}
+void h2muAnalyzer::endJob(){}
```
Comment (jhgoh) [inline CatAnalyzer/plugins/h2muAnalyzer.cc]: Better to remove unused functions :-)
beginJob and endJob are not automatically called by EDAnalyzer.

**Comment by jshlee** [comment]
by keeping only the electron and muon patTriggerObjectStandAlones we save a lot of space

patTriggerObjectStandAlones_selectedPatTrigger__PAT. 12642.8 1827.79
reduces to
patTriggerObjectStandAlones_catTrigger__CAT. 2535.65 355.238

**Comment by jhgoh** [comment]
good to see trigger info without restriction :+1:

**Pull request merged**